go-controller/pkg/node/gateway_udn.go

```diff
@@ -267,11 +267,13 @@ func (udng *UserDefinedNetworkGateway) AddNetwork() error {
 		waiter := newStartupWaiterWithTimeout(waitForPatchPortTimeout)
 		readyFunc := func() (bool, error) {
 			if err := setBridgeNetworkOfPorts(udng.openflowManager.defaultBridge, udng.GetNetworkName()); err != nil {
-				return false, fmt.Errorf("failed to set network %s's openflow ports for default bridge; error: %v", udng.GetNetworkName(), err)
+				klog.V(3).Infof("Failed to set network %s's openflow ports for default bridge; error: %v", udng.GetNetworkName(), err)
```

Comment:
nice catch! this explains why we were hitting
```
E1122 17:27:54.004932   14129 node_network_controller_manager.go:193] Stopping node network controller manager, err=failed to start NAD controller: initial sync failed: failed to sync network test-namespace-36.user-defined-network-36: [node-network-controller-manager network manager]: failed to start network test-namespace-36.user-defined-network-36: failed to add network to node gateway for network test-namespace-36.user-defined-network-36 at node ovn-worker3: error waiting for node readiness: failed to set network test-namespace-36.user-defined-network-36's openflow ports for default bridge; error: failed while waiting on patch port "patch-breth0_test.namespace.36.user.defined.network.36_ovn-worker3-to-br-int" to be created by ovn-controller and while getting ofport. stderr: ovs-vsctl: no row "patch-breth0_test.namespace.36.user.defined.network.36_ovn-worker3-to-br-int" in table Interface
, error: exit status 1
```
so soon and didn't see retries...

Reply:
```
@dceara the total timeout is 30seconds right so we won't hold it up more than that correct?
```

Correct.